types/router.go

```diff
@@ -40,7 +40,7 @@ type Route struct {
 
 // NewRoute returns an instance of Route.
 func NewRoute(p string, h Handler) Route {
-	return Route{path: p, handler: h}
+	return Route{path: strings.TrimSpace(p), handler: h}
```

Comment:
I was wondering if I should do the `TrimSpace` in line 58, but on `r.path` or just move it here.
I move it here for a hygiene.

Reply:
This is OK 👍